.github/dependabot.yml

```diff
@@ -16,6 +16,8 @@ updates:
       - "6.15.z"
       - "6.14.z"
       - "6.13.z"
+    ignore:
+      - dependency-name: "wrapanapi"
```

Comment:
I guess you meant `selenium` here? `wrapanapi` is not a dependency of airgun :)

Reply:
Yes